pynapple/core/time_series.py

```diff
@@ -1015,7 +1016,102 @@ def __getitem__(self, key):
             index = np.array([index])
         return _initialize_tsd_output(self, output, time_index=index)
 
-    @add_base_docstring("time_diff")
+    @add_docstring("get_slice", _Base)
+    def get_slice(self, start, end=None, time_unit="s"):
+        """
+        Examples
+        --------
+        >>> import pynapple as nap
+        >>> import numpy as np
+        >>> tsdtensor = nap.TsdTensor(t = [0, 1, 2, 3], d = np.random.randint(0, 1, (4, 3, 3)))
+
+        >>> # slice over a range
```

Comment:
```suggestion
         Slice over a range
         
```
Have leading comments as non-code statement before the code block. It will format more nicely in the terminal and doc pages